From 463a9e6352741e7bb358f8b8d9d437c17658c4b1 Mon Sep 17 00:00:00 2001
From: XScreenSaver owners <xscreensaver-owner@fedoraproject.org>
Date: Thu, 7 Sep 2023 23:28:19 +0900
Subject: [PATCH 4/4] ffmpeg-out.c: include additional header file
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

When configure'ing with --with-record-animation using
ffmpeg 6.0, ffmpeg-out.c won't compile like:

../../hacks/ffmpeg-out.c:47:3: error: unknown type name ‘AVCodecContext’
../../hacks/ffmpeg-out.c:91:17: warning: implicit declaration of function ‘avcodec_receive_packet’ [-Wimplicit-function-declaration]

... and so on.

Include needed additional header.
---
 hacks/ffmpeg-out.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hacks/ffmpeg-out.c b/hacks/ffmpeg-out.c
index 0295d7e..e54e9e6 100644
--- a/hacks/ffmpeg-out.c
+++ b/hacks/ffmpeg-out.c
@@ -37,6 +37,7 @@
   /* No "diagnostic pop" because some macrose use c99 features. */
 #endif
 
+#include <libavcodec/avcodec.h>
 #include <libavformat/avformat.h>
 #include <libswscale/swscale.h>
 #include <libswresample/swresample.h>
-- 
2.41.0

